GoogleUtilities/AppDelegateSwizzler/GULAppDelegateSwizzler.m

```diff
@@ -682,6 +701,17 @@ + (void)notifyInterceptorsWithMethodSelector:(SEL)methodSelector
   }];
 }
 
+#if UISCENE_SUPPORTED
++ (void)handleSceneWillConnectToNotification:(NSNotification *)notification {
+  if (@available(iOS 13.0, tvOS 13.0, *)) {
+    if ([notification.object isKindOfClass:[UIScene class]]) {
+      UIScene *scene = (UIScene *)notification.object;
+      [GULAppDelegateSwizzler proxySceneDelegate:scene];
```

Comment:
nit: I usually prefer adding suffix like `IfNeeded` to the method names if the action may or may not happen, e.g. `proxySceneDelegateIfNeeded:`. It makes the code a bit easier to read without looking at the method implementation. Though it's up to you, feel free to keep it as is.

Reply:
Yeah, suffix added.